src/cc/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set_target_properties(bcc-static PROPERTIES OUTPUT_NAME bcc)
 # BPF is still experimental otherwise it should be available
 #llvm_map_components_to_libnames(llvm_libs bpf mcjit irreader passes)
 llvm_map_components_to_libnames(llvm_libs bitwriter bpfcodegen irreader linker
-  mcjit objcarcopts option passes x86codegen)
+  mcjit objcarcopts option passes x86codegen powerpccodegen)
```

Comment:
Sure. I just tested with 'nativecodegen' and it actually works. It would also be appropriate for other architectures as well. I'll send a new pull request with this change. Thanks.